README.md

```diff
@@ -150,13 +150,20 @@ ui5lint --details
 
 #### `--format`
 
-Choose the output format. Currently, `stylish` (default), `json` and `markdown` are supported.
+Choose the output format. Currently, `stylish` (default), `json` and `markdown` are supported.  
```

Comment:
```suggestion
Choose the output format. Currently, `stylish` (default), `json`, and `markdown` are supported.  
```

Reply:
Sorry but is there actually a comma before and needed?

Reply:
Yes, please.